Podcast/SeriesDetailHeaderView.swift

```diff
@@ -64,145 +62,219 @@ class SeriesDetailHeaderView: UIView {
         infoView.backgroundColor = .offWhite
         infoView.clipsToBounds = true
         
-        imageView = ImageView(frame: CGRect(x: padding, y: padding, width: imageHeight, height: imageHeight))
+        backgroundImageView = ImageView()
```

Comment:
hmmm .. usually ImageView's initialized without a frame end up throwing an error, have you tested this?